packages/ui/styles/typography/ubuntu-mono.css

```diff
@@ -0,0 +1,31 @@
+@font-face {
+  font-family: 'Ubuntu Mono';
+  font-style: normal;
+  font-display: swap;
+  font-weight: 400;
+  src: url('./fonts/UbuntuMono-Regular.ttf') format('ttf');
```

Comment:
I would only include woff2
https://stackoverflow.com/questions/11002820/why-should-we-include-ttf-eot-woff-svg-in-a-font-face